src/components/FeedActivity.jsx

```diff
@@ -0,0 +1,96 @@
+import { Avatar,  Divider, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+      width: "100%"
+  },
+    avatar: {
+    marginTop: theme.spacing(2),
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(2)
+  },
+  name: {
+    marginBottom: theme.spacing(1),
+    fontSize: 14
+  },
+  stations: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  date: {
+    marginTop: theme.spacing(0),
+    marginLeft: theme.spacing(3),
+    color: theme.palette.grey[600],
+  },
+  vertDivider: {
+    backgroundColor: theme.palette.primary.main,
+    width: "2px",
+    marginLeft: theme.spacing(3),
+    marginTop: theme.spacing(1)
+  },
+  time: {
+    marginTop: theme.spacing(0),
+    marginBottom: theme.spacing(0),
+    color: theme.palette.grey[600],
+  },
+  container: {
+    display: "flex",
+  },
+  divider: {
+      width: "100%",
+      left: theme.spacing(2)
+  },
+  duration: {
+    color: theme.palette.grey[600],
+    marginLeft: theme.spacing(2)
+  },
+  activityExcercices: {
+    display: "flex",
+    flexDirection: "column",
+    marginLeft: theme.spacing(1),
+    marginTop: theme.spacing(1)
+  },
+  number: {
+    marginRight: theme.spacing(2)
+  }
+}));
+
+function FeedActivity() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.container}>
+          <Avatar
+            className={classes.avatar}
+            alt="Anette Stark"
+            src="https://images.sportbuzzer.de/v1/photos/raw/dsc04610/regular-16-9"
+          />
+          <p className={classes.name}><strong>Name</strong> <br/> x Stationen, x Übungen </p>
+        </div>
+        <div className={classes.container}>
+            <Divider orientation="vertical" flexItem className={classes.vertDivider} />
+            <span className={classes.activityExcercices}>
+                <Typography variant="caption">
+                    <span className={classes.number}>1</span>
+                    20 Liegestütze
+                    <span className={classes.duration}>5,4 Min</span>
```

Comment:
```suggestion
                    <span className={classes.duration}> 5,4 Min</span>
```
Mit dem Abstand haben wir ein Margin weniger :D